### ISSUE TITLE
Onboard & Offboard: Product

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/guides#92]
### Overview
We need to have a checklist of items to do when a product manager onboard and offboards so that the process can be consistent and fast.

### Date Updated
(insert date you created this issue and keep updating the date each time the org template change are integrated)

### Action Items
- [ ] Check to see if the org template for this process has new steps in it, and if it does, update this  issue and its template with content and add the last date you updated the template below
- [ ] Copy template to comment below
- [ ] Add name of person
- [ ] Check off as done

```
#### Onboard
- [ ] Slack channel membership
  - [ ] [insert name of channel]
  - [ ] [insert name of 2nd channel]
- [ ] Add to Google calendar invites
   - [ ] [DAY - PM meeting]
   - [ ] [DAY - BA meeting]
   - [ ] [DAY - UX Research meeting]
   - [ ] [DAY - All-team meeting]
   - [ ] [DAY - Design team meeting]
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roster
  - [ ] Check for 2FA and note in roster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [REPLACE with name of primary vault]
       - [ ] [REPLACE with name of secondary vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] [REPLACE with name of primary email]
   - [ ] [REPLACE with name of secondary email]
- [ ] Social Media Accounts with role assignment
   - [ ] Facebook
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite
```
```
#### Offboard
- [ ] Remove from Google calendar invite
   - [ ] [DAY - PM meeting]
   - [ ] [DAY - BA meeting]
   - [ ] [DAY - UX Research meeting]
   - [ ] [DAY - All-team meeting]
   - [ ] [DAY - Design team meeting]
- [ ] WIKI
  - [ ] move to new place on team page
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] [REPLACE with name of primary vault]
       - [ ] [REPLACE with name of secondary vault]
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] demote member to read only team and remove from write team
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ] [REPLACE with name of primary email]
   - [ ] [REPLACE with name of secondary email]
- [ ] Remove from Social Media Accounts with role assignment
   - [ ] Facebook
```

### Resources/Instructions
- [1password.com](https://1password.com/)
- [REPLACE with Google Drive Link]
- [REPLACE with Team Roster]
- Check public and 2FA at [people page](https://github.com/orgs/hackforla/people)
- [REPLACE with GitHub Team URL]
   - [base (read)]
   - [write]
   - [maintain]
   - [triage]
   - [admin]
   - [other]
- WIKI team page: https://github.com/hackforla/[Replace with name of your project]/wiki/The-Team
- [Replace with name of your project Figma]
- Shortcut for adding a profile to the desktop:  chrome://settings/manageProfile
- Tutorials
  - [Forward emails to team members](https://github.com/hackforla/product-management/wiki/Forward-emails-to-team-members)
---
[org template](https://raw.githubusercontent.com/hackforla/.github/main/.github/ISSUE_TEMPLATE/onboard---offboard--product.md) updated 2022-02-23
[Changelog](https://github.com/hackforla/product-management/wiki/Onboard-Offboard-Changelog)

**Comment by ExperimentsInHonesty** [comment]
#### Offboard Emmanuel kingojem 2022-02-25
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting
   - [ ] Tuesday - New Members meeting
   - [ ] Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] demote member to read only team and remove from write team
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by ExperimentsInHonesty** [comment]
#### Offboard olubabzy
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] demote member to read only team and remove from write team
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Offboard Ronny
- [x] Remove from Google calendar invite
   - [x] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [x] WIKI
  - [x] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [x] Google Drive
   - [x] demote to viewer
- [x] Roster
  - [x] mark as inactive
- [x] GitHub
     - [x] demote member to read only team and remove from write team
- [ ] Figma
  - [ ] remove from team
- [x] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [x] Remove from Social Media Accounts with role assignment  - not yet
   - [x] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Offboard Tunji @Headies 
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [x] Google Drive
   - [ ] demote to viewer
- [x] Roaster
  - [x] mark as inactive
- [x] GitHub
     - [x] demote member to read only team and remove from write team
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [x] Remove from Social Media Accounts with role assignment  - not yet
   - [x] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Offboard Aeding @AedingM  
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roaster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] demote member to read only team and remove from write team
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Offboard @Funmikee https://github.com/hackforla/guides/issues/93
- [ ] Close their onboarding issue and link here
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation)
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet
```

**Comment by SheIsBukki** [comment]
#### Offboard @Toluadekojo https://github.com/hackforla/guides/issues/83
- [ ] Close their onboarding issue and link here https://github.com/hackforla/guides/issues/83
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation)
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Onboard
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roaster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roaster
  - [ ] Check for 2FA and note in roaster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by SheIsBukki** [comment]
#### Offboard https://github.com/hackforla/guides/issues/106
- [ ] Close their onboarding issue and link here
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Wednesday - New Members meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Onboard @isaacmurna 
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roaster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roaster
  - [ ] Check for 2FA and note in roaster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by SheIsBukki** [comment]
#### Offboard https://github.com/hackforla/guides/issues/94
- [ ] Close their onboarding issue and link here https://github.com/hackforla/guides/issues/94
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by SheIsBukki** [comment]
#### Onboard @seannwvong 
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roaster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roaster
  - [ ] Check for 2FA and note in roaster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by edwardsarah** [comment]
#### Offboard Murna Isaac #111 
- [x] Close their onboarding issue and link here #111 
   - [x] Review all guide issues they are assigned to and remove them from those 
     - [x] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [x] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [x] Remove from Google calendar invite
   - [x] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [x] Figma
  - [x] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [x] Remove from Social Media Accounts with role assignment  - not yet
   - [x] Facebook - not yet

**Comment by edwardsarah** [comment]
#### Offboard Damilare Ogunsanya
- [x] Close their onboarding issue and link here #102 
   - [x] Review all guide issues they are assigned to and remove them from those 
     - [x] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [x] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [x] Remove from Google calendar invite
   - [x] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [x] Google Drive
   - [x] demote to viewer
- [x] Roster
  - [x] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [x] Figma
  - [x] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by edwardsarah** [comment]
#### Offboard Esther Adeagbo
- [x] Close their onboarding issue and link here #101 
   - [x] Review all guide issues they are assigned to and remove them from those 
     - [x] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [x] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [x] Remove from Google calendar invite
   - [x] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by edwardsarah** [comment]
@SheIsBukki I think I've done what I can in the offboarding of the team members listed above, but I'm not sure how to remove people from 1password (or check if they're there in the first place) or confirm their github permissions. Can you help with this?

**Comment by edwardsarah** [comment]
#### Offboard Prakhar
- [x] Close their onboarding issue and link here #100 
   - [x] Review all guide issues they are assigned to and remove them from those 
     - [x] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [x] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [x] Remove from Google calendar invite
   - [x] Monday - Team meeting, Friday - Team meeting
- [x] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [x] Roster
  - [x] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [x] Figma
  - [x] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by edwardsarah** [comment]
@SheIsBukki I'm not a manager of the Guides drive so I can't demote Prakhar to viewer, so if you could do so that would be great.

**Comment by edwardsarah** [comment]
#### Offboard Bukki
- [ ] Close their onboarding issue and link here
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by edwardsarah** [comment]
@ExperimentsInHonesty I can't remove Bukki from the google drive because I'm a content manager, not a manager, of the Guides google drive.

**Comment by edwardsarah** [comment]
#### Onboard
- [x] Slack channel membership
  - [x] ```#guides```
- [x] Add to Google calendar invites
   - [x] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [x] Google Drive
   - [x] Add to drive as Manager
- [x] Roaster
  - [x] Send link
  - [x] Team member adds themselves
  - [ ] Check for Public and note in roaster
  - [ ] Check for 2FA and note in roaster
- [x] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [x] Add to read vault
        - [x] Change membership to maintainer
- [x] Figma
  - [x] invite
  - [x] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by edwardsarah** [comment]
#### Onboard
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roaster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roaster
  - [ ] Check for 2FA and note in roaster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by ExperimentsInHonesty** [comment]
#### Onboard Claire W. 2023-10-20
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roaster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roaster
  - [ ] Check for 2FA and note in roaster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by edwardsarah** [comment]
#### Onboard Ayma Rahman
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roster
  - [ ] Check for 2FA and note in roster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by edwardsarah** [comment]
@ExperimentsInHonesty 
- How do I check for 2FA and public? I tried looking at GitHub's guidance on this issue but it doesn't reflect my experiences navigating the HfLA organization on GitHub
- Unable to add Ayma to guides-maintain, can we sort this out on Friday?
- Roster waiting for information on phone number, time zone, skills, availability, and linkedin
- Waiting on acceptance of email address forwarding to add filter
-

**Comment by edwardsarah** [comment]
#### Onboard Mary Gabrielian
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roster
  - [ ] Check for 2FA and note in roster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by edwardsarah** [comment]
#### Onboard Jeff
- [ ] Slack channel membership
  - [ ] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] Add to team page on Wiki
- [ ] Google Drive
   - [ ] Add to drive as Manager
- [ ] Roster
  - [ ] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roster
  - [ ] Check for 2FA and note in roster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by edwardsarah** [comment]
#### Offboard Mary
- [ ] Close their onboarding issue and link here
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by edwardsarah** [comment]
#### Offboard Ayma
- [ ] Close their onboarding issue and link here
   - [ ] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [ ] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Friday - Team meeting
- [ ] WIKI
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer
- [ ] Roster
  - [ ] mark as inactive
- [ ] GitHub
     - [ ] confirm that user is on the read only team
     - [ ] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [ ] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by ExperimentsInHonesty** [comment]
#### Onboard Jesus 2024-06-07
- [x] Slack channel membership
  - [x] ```#guides```
- [x] Add to Google calendar invites
   - [ ] ~Monday - Team meeting~ currently on hold
   - [x] Friday - Team meeting
- [x] Add to team page on Wiki
- [x] Google Drive
   - [x] Add to drive as Manager
- [x] Roster
  - [x] Send link
  - [x] Team member adds themselves
  - [x] Check for Public and note in roster
- [x] GitHub
     - [x] Add to team write team
        - [x] Change membership to maintainer
     - [x] Add to read vault
        - [x] Change membership to maintainer
- [x] Figma
  - [x] invite
  - [x] acceptance
- [x] 1password 
     - [x] invitation
     - [x] acceptance
     - [x] add to vaults
       - [x] [Guides vault]
       - [x] Zoom vault
- [x] Add to email address(s)
   - [x] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [x] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite
- [ ] Update the hackforla.org website team 
- [ ] Add PM Admin Access on VRMS

**Comment by edwardsarah** [comment]
#### Onboard France
- [x] Slack channel membership
  - [x] ```#guides```
- [x] Add to Google calendar invites
   - [ ] ~Monday - Team meeting~ currently on hold
   - [x] Friday - Team meeting
- [x] Google Drive
   - [x] Add to drive as Manager
- [x] Roster
  - [x] Send link
  - [x] Team member adds themselves
  - [x] Check for Public and note in roster
- [x] GitHub
     - [x] Add to team write team
        - [x] Change membership to maintainer
     - [x] Add to read vault
        - [x] Change membership to maintainer
- [x] Figma
  - [x] invite
  - [x] acceptance
- [x] 1password 
     - [x] invitation
     - [x] acceptance
     - [x] add to vaults
       - [x] [Guides vault]
       - [x] Zoom vault
- [x] Add to team page on Wiki
- [x] Add to email address(s)
   - [x] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [x] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite
- [x] Update the hackforla.org website team 
- [x] Add PM Admin Access on VRMS

**Comment by edwardsarah** [comment]
#### Onboard Aditya
#### Onboard
- [x] Slack channel membership
  - [x] ```#guides```
- [x] Add to Google calendar invites
   - [ ] ~Monday - Team meeting~ currently on hold
   - [x] Friday - Team meeting
- [x] Google Drive
   - [x] Add to drive as Manager
   - [x] Add to knowledgebase-content drive as Manager
- [x] Roster
  - [x] Send link
  - [x] Team member adds themselves
  - [ ] Check for Public and note in roster
- [x] GitHub
     - [x] Add to team write team
        - [x] Change membership to maintainer
     - [x] Add to read vault
        - [x] Change membership to maintainer
- [x] Figma
  - [x] invite
  - [x] acceptance
- [ ] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [x] Add to team page on Wiki
- [x] Add to team page on website
- [x] Create New Issue to Add PM Admin Access on VRMS
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by fchan218** [comment]
### Offboard Sarah

#### Offboard
- [x] Close their onboarding issue and link here https://github.com/hackforla/guides/issues/108
   - [x] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [x] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [x] Remove from Google calendar invite
   - [ ] Monday - Team meeting, Friday - Team meeting
- [x] WIKI
  - [ ] move to new place on team page - don't have yet
  - [x] 1password setup
    - [x] remove from team vaults
       - [x] Guides vault
       - [x] Zoom vault
- [x] Google Drive
   - [x] demote to viewer
- [x] Roster
  - [x] mark as inactive
- [x] Update the hackforla.org website team 
- [x] Remove PM Admin Access on VRMS
- [x] GitHub
     - [x] confirm that user is on the read only team, (as a member not a maintainer)
     - [x] remove from write team (only after read team confirmation) and indicate in the roster
- [x] Figma
  - [ ] remove from team
- [x] Help them delete the desktop shortcut to email address if they made one.
- [x] Remove from email address(s)
   - [ ]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet

**Comment by ExperimentsInHonesty** [comment]
#### Onboard 2024-11-12 Kevin T
intentionally didn't give him all permissions.  Going to work on guides using technical writing skills
- [x] Slack channel membership
  - [x] ```#guides```
- [x] Add to Google calendar invites
   - [ ] ~Monday - Team meeting~ currently on hold
   - [x] Friday - Team meeting
- [x] Google Drive
   - [x] Add to Guides drive drive as Manager
   - [x] Add to knowledgebase-content drive as Manager
- [ ] Roster
  - [x] Send link
  - [x] Team member adds themselves
  - [x] Check for Public and note in roster
- [x] GitHub
     - [x] Add to team write team
        - [ ] Change membership to maintainer
     - [x] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [x] 1password 
     - [x] invitation
     - [x] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to team page on Wiki
- [ ] Update the hackforla.org website team 
- [ ] Add PM Admin Access on VRMS
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by ExperimentsInHonesty** [comment]
#### Onboard 2025-03-05 Emmanuel
- [ ] Slack channel membership
  - [x] ```#guides```
- [ ] Add to Google calendar invites
   - [ ] ~Monday - Team meeting~ currently on hold
   - [x] Friday - Team meeting
- [ ] Google Drive
   - [x] Add to Guides drive as Manager
   - [x] Add to knowledgebase-content drive as Manager
- [ ] Roster
  - [x] Send link
  - [ ] Team member adds themselves
  - [ ] Check for Public and note in roster
- [ ] GitHub
     - [x] Add to team write team
        - [ ] Change membership to maintainer
     - [x] Add to read vault
        - [x] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [ ] 1password 
     - [x] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [ ] [Guides vault]
       - [ ] Zoom vault
- [ ] Add to team page on Wiki
- [ ] Update the hackforla.org website team 
- [ ] Add PM Admin Access on VRMS
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite

**Comment by fchan218** [comment]
#### Onboard 2025-03-21 Yashwanth Rongali 
- [x] Slack channel membership
  - [x] ```#guides```
- [x] Add to Google calendar invites
   - [ ] ~Monday - Team meeting~ currently on hold
   - [x] Friday - Team meeting
- [x] Google Drive
   - [x] Add to Guides drive as Manager
   - [x] Add to knowledgebase-content drive as Manager
- [x] Roster
  - [x] Send link
  - [x] Team member adds themselves
  - [ ] Check for Public and note in roster
- [ ] GitHub
     - [ ] Add to team write team
        - [ ] Change membership to maintainer
     - [ ] Add to read vault
        - [ ] Change membership to maintainer
- [ ] Figma
  - [ ] invite
  - [ ] acceptance
- [x] 1password 
     - [ ] invitation
     - [ ] acceptance
     - [ ] add to vaults
       - [x] [Guides vault]
       - [x] Zoom vault
- [ ] Add to team page on Wiki
- [ ] Update the hackforla.org website team 
- [ ] Add PM Admin Access on VRMS
- [ ] Add to email address(s)
   - [ ] guides@hackforla.org
- [ ] Social Media Accounts with role assignment - not yet
   - [ ] Facebook - not yet
- [ ] Train how to 
   - [ ] login to team account
   - [ ] setup chrome shortcut for adding to desktop
   - [ ] use zoom spreadsheet, vault and accounts
   - [ ] calendar invite
- [ ] Add new PM to VRMS [VRMS](https://vrms.io/)
    - [ ] send them the link 
       ```
       https://vrms.io
       ```
    - [ ] they setup account
 - [ ] Add the following text to the PM CoP agenda in a comment and customize it for the new PM
    ```
    Add [Person's Name] to [Project Name]
    - [ ] add to PM CoP agenda https://github.com/hackforla/product-management/issues/386
    - [ ] add to VRMS
    - [ ] add to 1password
    ```
    - [ ] Confirm access granted
 - Add PM to [Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg/edit?gid=2032152864#gid=2032152864)
    - [ ] send link
    - [ ] they add themselves
    - [ ] confirm all fields except for gray columns filled out

**Comment by fchan218** [comment]
#### Offboard Aditya 2025-04-05
- [ ] Close their onboarding issue and link here
   - [x] Review all guide issues they are assigned to and remove them from those 
     - [ ] Review their documents where they recorded their findings, to make sure we addressed or otherwise noted their observations
      - [ ] Reassign guide issues to team member(s), or add the Adopt Me label, or close the issues as necessary
- [x] Remove from Google calendar invite
   - [x] Monday - Team meeting, Friday - Team meeting
- [x] WIKI - moved to former team members
  - [ ] move to new place on team page - don't have yet
  - [ ] 1password setup
    - [ ] remove from team vaults
       - [ ] Guides vault
       - [ ] Zoom vault
- [ ] Google Drive
   - [ ] demote to viewer - done for Guides and knowledgebase-content but need to change to viewer for Product-Management CoP
- [x] Roster
  - [x] mark as inactive
- [x] Update the hackforla.org website team: https://github.com/hackforla/website/issues/8050
- [x] Remove PM Admin Access on VRMS
- [x] GitHub
     - [x] confirm that user is on the read only team, (as a member not a maintainer)
     - [x] remove from write team (only after read team confirmation) and indicate in the roster
- [ ] Figma
  - [ ] remove from team
- [ ] Help them delete the desktop shortcut to email address if they made one.
- [x] Remove from email address(s)
   - [x]  guides@hackforla.org
- [ ] Remove from Social Media Accounts with role assignment  - not yet
   - [ ] Facebook - not yet
